python/triton/runtime/jit.py

```diff
@@ -646,6 +658,12 @@ def _pack_args(self, backend, kwargs, bound_args, specialization, options):
         # constexprs
         constexprs = find_paths_if(sigvals, lambda _, val: val == "constexpr")
         constexprs = {path: get_iterable_path(list(bound_args.values()), path) for path in constexprs}
+        for key, value in constexprs.items():
+            if value.__class__.__name__ == "constexpr":
+                # Unwrap the constexpr to get its value; this helps to support serialization of the
+                # specialization data to JSON
+                constexprs[key] = value.value
```

Comment:
Sorry, looks like this regressed the kernel launch benchmarks. Perhaps you could do this unwrapping in `serialize_specialization_data` instead.

Reply:
Right, makes sense. Lets revert to your original PR then.